package.json

```diff
@@ -41,6 +43,7 @@
     "graphql-tag": "^2.11.0",
     "jsonwebtoken": "^8.5.1",
     "knex": "^0.21.5",
+    "luxon": "^1.25.0",
```

Comment:
@peter-asztalos is this a package for working with dates? What is the difference of using moment.js. I see it is under the same organisation but never used luxon before.

Reply:
[This](https://momentjs.com/docs/#/-project-status/) describes the status of moment.js very well, I think luxon can be describes as a possible successor of moment.js

Reply:
Good to hear and read that. I didn't know they have started to move over luxon. Now I have learned something as well 👨‍🔬 🕺